core/tests/unittests/metrics/test_metrics.py

```diff
@@ -0,0 +1,156 @@
+from math import isclose
+
+import numpy as np
+import pytest
+
+from autogluon.core.constants import BINARY, MULTICLASS, REGRESSION
+from autogluon.core.metrics import METRICS, Scorer
+
+
+BINARY_METRICS = list(METRICS[BINARY].keys())
+MULTICLASS_METRICS = list(METRICS[MULTICLASS].keys())
+REGRESSION_METRICS = list(METRICS[REGRESSION].keys())
+
+
+def test_metric_exists():
+    """
+    Ensure all expected metrics are present and no unexpected metrics are present
+    """
+    expected_metrics_and_aliases = {
```

Comment:
It is because those metrics are present in more than 1 problem type. It is due to how I created the list to begin with by for looping through METRICS. Good catch, I'll make it so I check each problem type individually, this will actually be more comprehensive.

Reply:
updated to check each problem type individually